crates/wallet/src/descriptor/error.rs

```diff
@@ -37,7 +37,7 @@ pub enum Error {
     /// Error during base58 decoding
     Base58(bitcoin::base58::Error),
     /// Key-related error
-    Pk(bitcoin::key::Error),
+    Pk(bitcoin::key::ParsePublicKeyError),
```

Comment:
I'm undecided if we need this `Pk` variant

Reply:
What are your thoughts about it?

Reply:
afaik it's not being used anywhere